src/Reflection/Adapter/ReflectionMethod.php

```diff
@@ -313,7 +313,7 @@ public function isDestructor()
     public function getClosure($object = null)
     {
         try {
-            $this->betterReflectionMethod->getClosure($object);
+            return $this->betterReflectionMethod->getClosure($object);
```

Comment:
Looks like we never tested this - that's needed here

Reply:
Yeah, this needs a test addition too

Reply:
Test added in 74ba5922c6910cb7954b09703f5086c2b29357d7

Reply:
Thanks!